Library/Homebrew/keg_relocate.rb

```diff
@@ -133,9 +133,16 @@ def replace_placeholders_with_locations(files, skip_linkage: false)
 
   def openjdk_dep_name_if_applicable
     deps = runtime_dependencies
+    deps ||= to_formula.deps # the tab is missing, so fall back to the formula
```

Comment:
Maybe this should be in the `runtime_dependencies` method. Also, is this going to also include `:build` dependencies?

Reply:
Actually, if we want this to return the same thing as `runtime_dependencies`, we probably want `to_formula.recursive_dependencies`. But then we probably have to deal with `TapFormulaUnavailableError`.